Index: VideoStore_Refactoring/src/ficha3/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ficha3;\r\n\r\n/***********************************************************\r\n * Filename: Main.java\r\n * @author fba 6 de Mai de 2013\r\n ***********************************************************/\r\npublic abstract class Main\r\n{\r\n\r\n\t/***********************************************************\r\n\t * @param args\r\n\t ***********************************************************/\r\n\tpublic static void main(String[] args)\r\n\t{\r\n\t\tCustomer who = new Customer(\"Barack Obama\");\r\n\t\tMovie m1 = new Movie(\"Life of Amalia\", Movie.Code.REGULAR);\r\n\t\tMovie m2 = new Movie(\"Peter Pan\", Movie.Code.CHILDRENS);\r\n\t\tMovie m3 = new Movie(\"Donna del Lago\", Movie.Code.NEW_RELEASE);\r\n\r\n\t\twho.addRental(new Rental(m1, 1));\r\n\t\twho.addRental(new Rental(m2, 2));\r\n\t\twho.addRental(new Rental(m3, 3));\r\n\t\t\r\n\t\tSystem.out.println(who.statement());\r\n\t}\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/VideoStore_Refactoring/src/ficha3/Main.java b/VideoStore_Refactoring/src/ficha3/Main.java
--- a/VideoStore_Refactoring/src/ficha3/Main.java	(revision 96aaf822693f0ce5c02debbe2246842a6aad844c)
+++ b/VideoStore_Refactoring/src/ficha3/Main.java	(date 1761931789142)
@@ -1,5 +1,18 @@
 package ficha3;
 
+import v9.ChildrensPrice;
+import v9.Customer;
+import v9.Movie;
+import v9.NewReleasePrice;
+import v9.Price;
+import v9.RegularPrice;
+import v9.Rental;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+
 /***********************************************************
  * Filename: Main.java
  * @author fba 6 de Mai de 2013
@@ -7,21 +20,29 @@
 public abstract class Main
 {
 
-	/***********************************************************
-	 * @param args
-	 ***********************************************************/
-	public static void main(String[] args)
-	{
-		Customer who = new Customer("Barack Obama");
-		Movie m1 = new Movie("Life of Amalia", Movie.Code.REGULAR);
-		Movie m2 = new Movie("Peter Pan", Movie.Code.CHILDRENS);
-		Movie m3 = new Movie("Donna del Lago", Movie.Code.NEW_RELEASE);
+    /***********************************************************
+     * @param args
+     * @throws IOException
+     ***********************************************************/
+    public static void main(String[] args) throws IOException
+    {
+        v9.Price regularPrice = new RegularPrice();
+        v9.Price childrensPrice = new ChildrensPrice();
+        Price newReleasePrice = new NewReleasePrice();
 
-		who.addRental(new Rental(m1, 1));
-		who.addRental(new Rental(m2, 2));
-		who.addRental(new Rental(m3, 3));
-		
-		System.out.println(who.statement());
-	}
+        v9.Customer who = new Customer("Barack Obama");
+        v9.Movie m1 = new v9.Movie("Life of Amalia", regularPrice);
+        v9.Movie m2 = new v9.Movie("Peter Pan", childrensPrice);
+        v9.Movie m3 = new Movie("Donna del Lago", newReleasePrice);
+
+        who.addRental(new v9.Rental(m1, 1));
+        who.addRental(new v9.Rental(m2, 2));
+        who.addRental(new Rental(m3, 3));
+        System.out.println( who.statement());
 
-}
+        PrintWriter html = new PrintWriter(new FileWriter("webPages/statement.html"));
+        html.println(who.htmlStatement());
+        html.close();
+    }
+
+}
\ No newline at end of file
